package-kafka-connect-storage-common/pom.xml

```diff
@@ -47,11 +47,6 @@
             <artifactId>kafka-connect-storage-partitioner</artifactId>
             <version>${project.version}</version>
         </dependency>
-        <dependency>
-            <groupId>io.confluent</groupId>
-            <artifactId>kafka-connect-storage-hive</artifactId>
-            <version>${project.version}</version>
-        </dependency>
```

Comment:
Am I understanding it that we're removing this as a dependency, but we're keeping the `kafka-connect-storage-hive` module and code to only be including it in other modules (the HDFS connector) specifically via a dependency?

Reply:
I think from https://github.com/confluentinc/kafka-connect-hdfs/pull/429 the answer to this is "yes".

Reply:
Indeed. This is removed from the packaging of Confluent platform, but will still be published in our public maven repo (nexus) for any other module, like the hdfs connector, to import it as a dependency